python/paddle/fluid/layers/detection.py

```diff
@@ -683,6 +684,10 @@ def prior_box(input,
             Default:[0.1, 0.1, 0.2, 0.2].
        flip(bool): Whether to flip aspect ratios. Default:False.
        clip(bool): Whether to clip out-of-boundary boxes. Default: False.
+       def_ord(bool): Whether to output prior box in default order. If set True,
+            this layer will output in the order of [min, various_ar, max]
+            otherwise, this layer will output [min, max, various_ar]
+            Default: True.
```

Comment:
Maybe  def_ord  ->  min_max_aspect_ratios_order  ？

If set True,  the output prior box is in order of [min, max, aspect_ratios], which is consistent with Caffe. Please note,  this order affects the weights order of convolution layer followed by and does not affect the fininal detection results.

Please also update the comments in C++ doc.

Reply:
Please also update the argument of 

https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/layers/detection.py#L766

Reply:
Done